DB_Schema/leaderboard.md

```diff
@@ -55,5 +114,47 @@ Note: make sure to use `__` double underscore
 ## To Create Tables Run a SQL file like
 
 ```
-mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
+$ mysql -u root -p --no-data dbname < create_leaderboard_tables.sql
```

Comment:
Thanks!
Also, is `create_leaderboard_tables.sql` in Git? Please link to it here.

Reply:
I added the cd command to be clear

Reply:
Great, thanks :-)